pkg/rancher-desktop/integrations/pathManagerImpl.ts

```diff
@@ -92,11 +118,11 @@ export class RcFilePathManager implements PathManager {
       return manageLinesInFile(rcPath, [pathLine], desiredPresent);
     }));
 
-    mainEvents.invoke('diagnostics-trigger', 'RD_BIN_IN_BASH_PATH');
-    mainEvents.invoke('diagnostics-trigger', 'RD_BIN_IN_ZSH_PATH');
+    await mainEvents.invoke('diagnostics-trigger', 'RD_BIN_IN_BASH_PATH');
+    await mainEvents.invoke('diagnostics-trigger', 'RD_BIN_IN_ZSH_PATH');
```

Comment:
I'm not sure we need to wait for these to run to completion; we could at least make them run in parallel (using `Promise.all`), I think?